tripal_chado/tests/src/Functional/Services/ChadoTripalPublishTest.php

```diff
@@ -414,27 +414,27 @@ public function testChadoTripalPublishService() {
 
     $this->checkFieldItem('organism', 'organism_genus', 1,
         ['record_id' => $organism_id2],
-        ['bundle' => 'organism', 'entity_id' => 2, 'value' => NULL]);
+        ['bundle' => 'organism', 'entity_id' => 2, 'value' => '']);
 
     $this->checkFieldItem('organism', 'organism_species', 1,
         ['record_id' => $organism_id2],
-        ['bundle' => 'organism', 'entity_id' => 2, 'value' => NULL]);
+        ['bundle' => 'organism', 'entity_id' => 2, 'value' => '']);
 
     $this->checkFieldItem('organism', 'organism_abbreviation', 1,
         ['record_id' => $organism_id2],
-        ['bundle' => 'organism', 'entity_id' => 2, 'value' => NULL]);
+        ['bundle' => 'organism', 'entity_id' => 2, 'value' => '']);
 
     $this->checkFieldItem('organism', 'organism_infraspecific_name', 0,
         ['record_id' => $organism_id2],
-        ['bundle' => 'organism', 'entity_id' => 2, 'value' => NULL]);
+        ['bundle' => 'organism', 'entity_id' => 2, 'value' => '']);
```

Comment:
Since the infraspecific name and type don't have a value in chado, shouldn't they not have an entry in the drupal field table? I suspect this will also fail when the token replacement issue gets fixed and if it doesn't then it probably should... shouldn't it?

Reply:
It is correct that they should not have an entry in the drupal field table.
Since we are passing `0` for $num_expected, this is testing that this is indeed the case 😀

Reply:
Ahhh! A comment to that effect would be really helpful!

Reply:
Comment added with commit https://github.com/tripal/tripal/pull/1952/commits/99d2724377c24622ac1b7a858394fed13bf6d98c